Sample Solutions/Viva Insights Query Analysis Automation/SQLscripts/MeetingQueryTbl.sql

```diff
@@ -0,0 +1,43 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[viva_insights_meeting]
+( 
+	[MeetingId] [nvarchar](4000)  NOT NULL,
```

Comment:
CREATE TABLE [dbo].[zCoryTest_delme2]
(
[MeetingId] nvarchar NOT NULL,
[StartTimestampUTC] datetime2 NOT NULL,
[EndTimestampUTC] datetime2 NOT NULL,
[Organizer_PersonId] nvarchar NOT NULL,
[Organizer_Organization] nvarchar NULL,
[Organizer_LevelDesignation] nvarchar NULL,
[Organizer_IsInternal] [bit] NULL,
[Attendees] [int] NULL,
**### PRIMARY KEY ([MeetingId],[StartTimestampUTC])**
)

Need a better way to do composite primary key
Result:

Warning! The maximum key length for a clustered index is 900 bytes. The index 'PK__zCoryTes__4918EBB662A27C48' has maximum length of 8008 bytes. For some combination of large values, the insert/update operation will fail.